packages/components/filter-bar/pipe.directive.ts

```diff
@@ -0,0 +1,47 @@
+import { AfterContentInit, Directive, inject, Injector, Input, ViewContainerRef } from '@angular/core';
+import { KbqPipe, KbqPipeTemplate, KbqPipeTypes } from './filter-bar.types';
+import { KBQ_PIPE_DATA } from './pipes/base-pipe';
+import { KbqPipeDateComponent } from './pipes/pipe-date';
+import { KbqPipeDatetimeComponent } from './pipes/pipe-datetime';
+import { KbqPipeMultiSelectComponent } from './pipes/pipe-multi-select';
+import { KbqPipeSelectComponent } from './pipes/pipe-select';
+import { KbqPipeTextComponent } from './pipes/pipe-text';
+
+@Directive({
+    standalone: true,
+    selector: '[kbqPipe]'
```

Comment:
чтобы сразу было понятно, к какому модулю относится директива
```suggestion
    selector: '[kbqFilterBarPipe]'
```

Reply:
Да не, иначе нужно будет переименовать остальные пайпы `(KbqPipeDateComponent, KbqPipeMultiSelectComponent)` и т.д. и фактически пользы от этого нет, вроде понятно, что пайпы используются только в контексте filter-bar и другие пайпы пока не планируем добавлять.

Посмотрел на reset, search и refresher, классы называются правильно, а файлы нет, переименую.